packages/xo-web/src/xo-app/proxies/deploy-proxy.js

```diff
@@ -228,7 +228,17 @@ const Modal = decorate([
   ),
 ])
 
-const deployProxy = ({ license, proxy }) => {
+const deployProxy = async ({ proxy }) => {
+  const licenses = await getLicenses({ productType: 'xoproxy' })
+  const license =
+    licenses &&
```

Comment:
What this test?

Reply:
License will be `undefined` if the user don't have licenses.

Reply:
AFAIK, it's a bad idea to have different types of returned value for a function…